pkg/scheduler/partition.go

```diff
@@ -585,10 +580,6 @@ func (pc *PartitionContext) AddNode(node *objects.Node, existingAllocations []*o
 
 	// Node is added update the metrics
 	metrics.GetSchedulerMetrics().IncActiveNodes()
-	log.Logger().Info("added node to partition",
-		zap.String("partitionName", pc.Name),
-		zap.String("nodeID", node.NodeID),
-		zap.String("partitionResource", pc.totalPartitionResource.String()))
```

Comment:
This one can be helpful as well, I think. Maybe at debug level is enough, but I don't think we should remove it.

Reply:
this has been moved to an INFO level message in the partition to standardise the way we log.